citre-lang-c.el

```diff
@@ -133,8 +138,10 @@
                    (list
                     `(filter ,(citre-core-filter 'input path 'suffix) +)))))
          ('member
-          (citre-core-sorter
-           `(filter ,(citre-core-filter-kind "member" tagsfile) +)))
+          (apply #'citre-core-sorter
+                 `(filter ,(citre-core-filter-kind "member" tagsfile) +)
+                 (when (citre-get-property 'callable-member symbol)
+                   (list `(filter ,(citre-core-filter 'typeref "(*)" 'substr) +)))))
```

Comment:
I think I know the `(*)` is a function pointer thing, but I don't know it well. Could you write a comment explaining it here? Of course you don't have to if it's a common knowledge to C programmers.